test/WebSites/BasicWebSite/Areas/Area1/Views/RemoteAttribute_Home/Details.cshtml

```diff
@@ -1,10 +1,7 @@
-﻿@model ValidationWebSite.Models.Person
+﻿@model BasicWebSite.Models.RemoteAttributeUser
 @{
-    object areaObject;
-    ViewContext.ActionDescriptor.RouteValueDefaults.TryGetValue("area", out areaObject);
-    var areaName = (areaObject as string) ?? "root";
-
-    ViewBag.Title = "Details in " + areaName + " area.";
+    Layout = "/Areas/Area1/Views/RemoteAttribute_Home/_Layout.cshtml";
+    ViewBag.Title = "Details in Area1 area.";
```

Comment:
Again, don't remove the area name calculation and use a relative path for the `Layout`.